html_app/model/CellTreatment.js

```diff
@@ -93,6 +93,7 @@ scb.CellTreatment = function scb_CellTreatment(data, context, parent) {
                 new_text = new_text.replace( regexp , resolve(value));
             });
             display_text = new_text == orig_template ? display_text : new_text;
+            display_text = display_text.replace(/(, *)+/g, ", ");
```

Comment:
This seems a little odd to remove the extra commas after the fact. Where does the display_text get constructed? Is there a way to omit the empty fields at that point?

Not a big deal. If it's too painful to change, I'm :+1: